WooCommerce/Classes/Authentication/Prologue/LoginProloguePageViewController.swift

```diff
@@ -27,7 +26,12 @@ final class LoginProloguePageViewController: UIPageViewController {
             setViewControllers([firstPage], direction: .forward, animated: false)
         }
 
-        addPageControl()
+        if pages.count > 1 {
+            addPageControl()
+        } else {
+            // Sets data source to `nil` to disable scrolling.
+            dataSource = nil
+        }
```

Comment:
Super nit:
Kindly consider extracting this into a private method.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/7291/commits/c7dd9ab4d249c7c36cd770a2275570c43fa9f76d